spec/features/idv/analytics_spec.rb

```diff
@@ -1071,65 +1071,332 @@ def wait_for_event(event, wait)
+          row = CSV.parse(
+            Reports::DailyDropoffsReport.new.tap { |r| r.report_date = Time.zone.now }.report_body,
+            headers: true,
+          ).first
+
+          Reports::DailyDropoffsReport::STEPS.each do |step|
+            expect(row[step].to_i).to(be > 0, "step #{step} was counted")
+          end
+        end
+      end
+
+      context 'proofing_device_profiling disabled' do
+        let(:proofing_device_profiling) { :disabled }
+        let(:threatmetrix) { false }
+        let(:threatmetrix_response) do
+          {
+            client: 'tmx_disabled',
+            success: true,
+            errors: {},
+            exception: nil,
+            timed_out: false,
+            transaction_id: nil,
+            review_status: 'pass',
+            account_lex_id: nil,
+            session_id: nil,
+            response_body: nil,
+          }
+        end
+
+        it 'records all of the events' do
+          aggregate_failures 'analytics events' do
+            happy_hybrid_path_events.each do |event, attributes|
+              expect(fake_analytics).to have_logged_event(event, attributes)
+            end
+          end
+        end
+      end
+    end
+    context 'GPO path' do
+      before do
+        sign_in_and_2fa_user(user)
+        visit_idp_from_sp_with_ial2(:oidc)
+        complete_welcome_step
+        complete_agreement_step
+        complete_hybrid_handoff_step
+        complete_document_capture_step
         complete_ssn_step
         complete_verify_step
-        fill_out_phone_form_ok('202-555-1212')
-        verify_phone_otp
+        enter_gpo_flow
+        complete_request_letter
         complete_enter_password_step(user)
-        acknowledge_and_confirm_personal_key
       end
-    end
 
-    it 'records all of the events' do
-      happy_mobile_selfie_path_events.each do |event, attributes|
-        expect(fake_analytics).to have_logged_event(event, attributes)
+      it 'records all of the events', allow_browser_log: true do
+        gpo_path_events.each do |event, attributes|
+          expect(fake_analytics).to have_logged_event(event, attributes)
+        end
+      end
+
+      context 'proofing_device_profiling disabled' do
+        let(:proofing_device_profiling) { :disabled }
+        let(:threatmetrix) { false }
+        let(:threatmetrix_response) do
+          {
+            client: 'tmx_disabled',
+            success: true,
+            errors: {},
+            exception: nil,
+            timed_out: false,
+            transaction_id: nil,
+            review_status: 'pass',
+            account_lex_id: nil,
+            session_id: nil,
+            response_body: nil,
+          }
+        end
+
+        it 'records all of the events', allow_browser_log: true do
+          gpo_path_events.each do |event, attributes|
+            expect(fake_analytics).to have_logged_event(event, attributes)
+          end
+        end
       end
     end
```

Comment:
please let me know if you see differently ... but i think we can delete this test as i do not think it is adding anything when we have the GPO test above

Reply:
the GPO is really far away form the FF and should not have any dependencies on the FF, so I don't think the test is adds any value.  for a moment I also questioned the IPP test added but at least that code is somewhat related to document capture and i like keeping it around. ultimately, since we will be removing the duplicate tests soon, I am fine with can keeping the GPO test if you feel strongly.

Reply:
@amirbey Let's leave it in. If needed, I can add a comment to clarify that this is duplicated

Reply:
@AShukla-GSA - revisiting this ... what is the value add? i'm seeing add'l tests also in verify info and it is unclear to me as to why we're adding code to our codebase and causing burden to our CI/CD process